packages/fstar/fstar.2025.02.06/opam

```diff
@@ -0,0 +1,44 @@
+opam-version: "2.0"
+maintainer: "taramana@microsoft.com"
+authors: [
+  "Nik Swamy <nswamy@microsoft.com>"
+  "Jonathan Protzenko <protz@microsoft.com>"
+  "Tahina Ramananandro <taramana@microsoft.com>"
+]
+homepage: "http://fstar-lang.org"
+license: "Apache-2.0"
+depends: [
+  "ocaml" {>= "4.14.0"}
+  "batteries"
+  "zarith"
```

Comment:
```suggestion
  "zarith" {>= "1.14"}
```

there are lower-deps failures (segfault) and this change is to try to pinpoint them